### PR TITLE
Fix #227 - Add link to Plugin SDK in "Extend SPIRE" page

### LINKED ISSUE #227: Extending page could use a link to the SPIRE Plugin SDK
The [SPIRE Plugin SDK](https://github.com/spiffe/spire-plugin-sdk) is the new one-stop-shop for plugin authors. Can we get a new (short) section added this page https://spiffe.io/docs/latest/planning/extending/ about authoring plugins with a link to the SDK?

Thanks, @dennisgove for the suggestion!

### PR DESCRIPTION
**Pull request by sanderson042**
Signed-off-by: Steve Anderson <steve.anderson@hpe.com>

### CODE REVIEW

**Review by azdagron — APPROVED** [review]
LGTM! Thanks for jumping on this @sanderson042 !

File: content/docs/latest/planning/extending.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
     - /spire/docs/extending
     - /docs/latest/spire/developing/extending
 ---
 
-SPIRE is highly extensible via a plugin framework that allows many core operations to be added and customized.
+SPIRE is highly extensible via a plugin framework that allows many core operations to be added and customized. This page describes the types of plugins available, how to integrate plugins while deploying SPIRE, and has a link to the SPIRE Plugin SDK for those interested in authoring a plugin.
+
+# Authoring SPIRE plugins
+
+Refer to the [Spire Plugin SDK](https://github.com/spiffe/spire-plugin-sdk) GitHub page for documentation and templates that you can use to code your own SPIRE plugins.
```
Comment (mchurichi) [inline content/docs/latest/planning/extending.md:16]:
```suggestion
Refer to the [SPIRE Plugin SDK](https://github.com/spiffe/spire-plugin-sdk) GitHub page for documentation and templates that you can use to code your own SPIRE plugins.
```

**Review by mchurichi — APPROVED** [review]
Thanks, @sanderson042 for jumping so quickly on this! LGTM, just left a minor suggestion.

File: content/docs/latest/planning/extending.md (hunk 1 above)
Comment (azdagron) [reply content/docs/latest/planning/extending.md:16]: Good catch, @mchurichi !

**Pull request merged**